framework/db/ActiveQuery.php

```diff
@@ -796,4 +789,89 @@ public function alias($alias)
         }
         return $this;
     }
+
+    /**
+     * Table names based on [[from]]
+     * @return string[] table names
+     * @throws \yii\base\InvalidConfigException
+     * @since 2.0.12
+     */
+    public function getTableNamesUsedInFrom()
+    {
```

Comment:
There are table names used in sub-queries and other cases. The method works only on `from` clause.